### PR TITLE
Release 3.5.2

### PR DESCRIPTION
**Pull request by harry-brown-cko**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to Frames Ios?
_Put an `x` in the boxes that apply_

* [ ] Bugfix (non-breaking change which fixes an issue)
* [ ] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

* [ ] Lint and unit tests pass locally with my changes
* [ ] I have added tests that prove my fix is effective or that my feature works
* [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: iOS Example Frame/Podfile.lock [hunk 1]
```diff
@@ -1,9 +1,9 @@
 PODS:
   - CheckoutEventLoggerKit (1.1.1)
-  - Frames (3.5.0):
+  - Frames (3.5.1):
```
Comment (deepesh-vasthimal-cko) [inline iOS Example Frame/Podfile.lock:3]: Should this be Frames 3.5.2 ?

File: iOS Example Frame/Podfile.lock [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
     - CheckoutEventLoggerKit (= 1.1.1)
     - PhoneNumberKit (= 3.3.3)
-  - Frames/Tests (3.5.0):
+  - Frames/Tests (3.5.1):
```
Comment (deepesh-vasthimal-cko) [inline iOS Example Frame/Podfile.lock:6]: Should this be Frames 3.5.2 ?

**Review by deepesh-vasthimal-cko — COMMENTED** [review]
Waiting for @sweeni-nalla-cko to test : https://checkout.atlassian.net/browse/PIMOB-877

**Review by deepesh-vasthimal-cko — APPROVED** [review]

File: iOS Example Frame/Podfile.lock (hunk 1 above)
Comment (harry-brown-cko) [reply iOS Example Frame/Podfile.lock:3]: These sample projects are just linked to the files in the current repo. This shouldn't be an issue

**Pull request merged**